src/Behat/Testwork/ServiceContainer/Configuration/ConfigurationTree.php

```diff
@@ -30,13 +30,12 @@ final class ConfigurationTree
      */
     public function getConfigTree(array $extensions)
     {
-        $tree = new TreeBuilder();
-        $root = $tree->root('testwork');
+        $root = new TreeBuilder('testwork');
 
         foreach ($extensions as $extension) {
-            $extension->configure($root->children()->arrayNode($extension->getConfigKey()));
+            $extension->configure($root->getRootNode()->children()->arrayNode($extension->getConfigKey()));
```

Comment:
this will be broken on older Symfony versions. You need to keep the `$root` variable, but defining it as either `$tree->getRootNode()` or `$tree->root('testwork')` depending on the version of the component (the detection should be based on `method_exists($tree, 'getRootNode')`